engine/runtime/src/main/java/org/enso/interpreter/epb/node/PyForeignNode.java

```diff
@@ -0,0 +1,25 @@
+package org.enso.interpreter.epb.node;
+
+import com.oracle.truffle.api.dsl.NodeField;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.interop.ArityException;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.interop.UnsupportedTypeException;
+import com.oracle.truffle.api.library.CachedLibrary;
+
+@NodeField(name = "foreignFunction", type = Object.class)
```

Comment:
Can you implement the `JsForeignNode` using `@NodeField`? Consistency matters for this, otherwise it might be seen as intentional or important.